test/controllers/CanPayWithinSixMonthsControllerSpec.scala

```diff
@@ -111,15 +111,35 @@ class CanPayWithinSixMonthsControllerSpec extends ItSpec with PegaRecreateSessio
         JourneyJsonTemplates.`Obtained Can Pay Within 6 months - no`(Origins.Epaye.Bta)(testCrypto)
       )
 
-      val result = controller.canPayWithinSixMonths(Epaye)(fakeRequest)
+      val result = controller.canPayWithinSixMonths(Epaye, None)(fakeRequest)
       testPageIsDisplayed(result, Some(false))
     }
 
+    "change the language cookie to english" in {
+      stubCommonActions()
+      EssttpBackend.CanPayWithinSixMonths.findJourney(testCrypto, Origins.Epaye.Bta)(
+        JourneyJsonTemplates.`Obtained Can Pay Within 6 months - no`(Origins.Epaye.Bta)(testCrypto)
+      )
+
+      val result = controller.canPayWithinSixMonths(Epaye, Some(English))(fakeRequest)
+      cookies(result).get("PLAY_LANG").map(_.value) shouldBe Some("en")
+    }
+
+    "change the language cookie to welsh" in {
```

Comment:
we could check that the language doesn't change if `None` is passed in as well perhaps like:
```scala
Languages.values.foreach{ lang => // there's may or may not be a values you can use
  val result = controller.endpoint(regime, fakeRequestWithLang(lang))
  cookies(result).get("blah").map(_.value) shouldBe Some(lang.code)
}
```